src/Data/Monoid/Additive.purs

```diff
@@ -0,0 +1,46 @@
+module Data.Monoid.Additive where
```

Comment:
If you really want to go crazy, you can also define `Alternative`, `MonadPlus`.

There's also `Foldable` and `Traversable`.

Same for `Multiplicative`.